service/build.gradle

```diff
@@ -169,7 +169,7 @@ jacocoTestReport {
         xml.enabled = true
         xml.destination = file("$buildDir/reports/jacoco/test/jacoco.xml")
     }
-    dependsOn test
+//    dependsOn test
```

Comment:
why is this commented out?

Reply:
whoops, fixed. I think our Jacoco setup is the root cause of unit tests running twice and I was investigating, but that's not for this branch.